pump/storage/storage.go

```diff
@@ -686,12 +688,22 @@ func (a *Append) doGCTS(ts int64) {
 			return
 		}
 
+		failpoint.Inject("l0-file-too-much", func() {
+			if l0TestLarge < l0Trigger {
+				l0TestLarge++
+			} else {
+				l0TestLarge -= l0Trigger
+			}
+			l0Num = l0TestLarge
+		})
+
 		if l0Num >= l0Trigger {
 			log.Info("wait some time to gc cause too many L0 file", zap.Int("files", l0Num))
 			if iter != nil {
 				iter.Release()
 				iter = nil
 			}
+			failpoint.Continue()
```

Comment:
here, when we switch on the failpoint, this line will be rewritten to `continue`, so we skip the `Sleep` in test. @suzaku

Reply:
I see. But we still need to validate that the behavior "LevelDB GC doesn't happen when there are too many l0 files".